extension/data/guiobject/guiobject.html

```diff
@@ -2,7 +2,8 @@
 <html>
 <head>
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-  <link rel="stylesheet" href="widget.css" type="text/css" />
+  <link rel="stylesheet" href="guiobject.css" type="text/css" />
+  <script type="application/javascript;version=1.8" src="guiobject.js"></script>
```

Comment:
So this element group is only fully displayed inside the toolbar but not in the sidebar nor customize dialog. Can we rename the related files to toolbar.(html, css, js)? It would better reflect where it is used.